### DISCUSSION TITLE
[Suggestion] Some ideas on how different games make warp mechanics much more deep

### DISCUSSION THREAD

**Discussion by Kaiser-Indrasil** [tsunamayo/Starship-EVO#4004] [Suggestions]
In EVE Online you must be going at least at 75% of your top speed in a very narrow corridor towards your target, so the speed effectively becomes your spool up period, in terms of gameplay mechanics.

If we had a "slowing ray", the spool up speed would automatically adjust.
There could also be an "interdiction ray" that the attacking party could tackle the target about to warp out.

You can also prevent a ship from warping by simply bumping into it, throwing it off the align trajectory.
There are interdiction bubbles which, if placed along your path, will force you out of warp.

In Elite: Dangerous there is a mid-warp interdiction mechanic that has you play a mini-game where you can escape the interdiction attempt.

E:D also features the "Speed up and align" mechanic that is present in EVE, but only for inter-system jumps.
For an intra-system jump, you only need to speed up cos it acts as lightcruise.

In Freespace the warp is its own dimension where the shields don't work and you can blow stuff up whose shields would normally be impervious to your attacks.

There also was a tactic of "shock jumping" where ships emerging from warp would maintain their momentum which they accumulated before the warp, letting them close in on their target in a matter of seconds to deliver a deadly alpha strike.

**Comment by Kaiser-Indrasil** [comment]
Freespace and EVE also feature jump nodes which are a network of fixed 2-point inter-system connections that let ships jump across distances they normally wouldn't be able to on their own because they are not equipped with warp cores, or the distance is too big.
These are strategic choke points that streamline blockades and are a big factor when devising logistics operations in the region.

**Comment by RockefellerDoctor** [comment]
There is a now-defunct early access game named Hellion which had a unique warp system that had you manually line up your jump while taking gravity curves into account.

**Comment by tsunamayo** [comment]
@Kaiser-Indrasil yep that all sounds like a solid idea, but first I want to make some progress with the combat, as one solution might emerge as more elegant than one other.
